drivers/ioexpander/tca64xx.c

```diff
@@ -254,6 +273,50 @@ static uint8_t tca64_config_reg(FAR struct tca64_dev_s *priv, uint8_t pin)
   return reg + (pin >> 3);
 }
 
+/****************************************************************************
+ * Name: tca64_pdenable_reg
+ *
+ * Description:
+ *  Return the address of the pu/pd enable register for the specified pin.
+ *
+ ****************************************************************************/
+
+static uint8_t tca64_pdenable_reg(FAR struct tca64_dev_s *priv, uint8_t pin)
+{
+  FAR const struct tca64_part_s *part = tca64_getpart(priv);
+  uint8_t reg = part->tp_puenable;
+
+  if (reg == TC64XX_INVALID_REG)
```

Comment:
If you do other way around;

if (reg === 0) return TC64XX_INVALID_REG;

This makes more sense. Now this  returns 0, which is a valid register address.... even though never pdenable register.

Reply:
done